Styles/Ortho.json

```diff
@@ -10,7 +10,7 @@
       "maxzoom": 16
     }
   },
-  "sprite": "https://israelhikingmap.github.io/VectorMap/Icons/publish/sprite",
+  "sprite": "https://israelhiking.osm.org.il/content/sprite/sprite",
```

Comment:
I wonder why this raster style has `sprite` and `glyphs`.

[MapLibre Style Spec](https://maplibre.org/maplibre-style-spec/sprite/) is not clear if they are a must .